src/server/service/user.ts

```diff
@@ -22,6 +21,26 @@ export default class UserService implements IUserService {
 
   constructor(loggerService: ILoggerService) {
     this.loggerService = loggerService;
+    this.createAdmin();
+  }
+
+  private async createAdmin(): Promise<void> {
+    const existentAdmin = await UserModel.findOne({ role: 'admin' });
+
+    if (existentAdmin) {
+      return this.loggerService.info('Admin user already exists');
+    }
+
+    try {
+      await this.create({
+        email: 'admin@gmail.com',
+        password: 'admin123',
+        role: 'admin',
+      });
```

Comment:
This conflicts with the implementation of `create` we have in #7 .

Reply:
I would implement a function in another place of the application (say `utils/bootstrap.ts`), have the services injected to such function and then create a user from there.

Reply:
I don't think this belongs to the `UserService` because this is a special process, part of the procedure of initializing the application and not a service consumed after initialization.

This should run one time only.